src/test/java/page/objects/LoginPage.java

```diff
@@ -0,0 +1,92 @@
+package page.objects;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class LoginPage {
+
+    WebDriver driver;
+    WebDriverWait wait;
+
+    public LoginPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(id = LoginPageLocators.USERNAME_ID)
+    @CacheLookup
+    WebElement username;
+
+    @FindBy(id = LoginPageLocators.PASSWORD_ID)
+    @CacheLookup
+    WebElement password;
+
+    @FindBy(id = LoginPageLocators.LOGIN_ID)
+    @CacheLookup
+    WebElement loginButton;
+
+    @FindBy(xpath = LoginPageLocators.ERROR_XPATH)
+    @CacheLookup
+    WebElement errorMessage;
+
+    @FindBy(className = LoginPageLocators.HAMBURGER_BUTTON_CLS)
+    @CacheLookup
+    WebElement hamburgerButton;
+
+    @FindBy(xpath = LoginPageLocators.LOGOUT_BUTTON_XPATH)
+    @CacheLookup
+    WebElement logoutButton;
+
+    public void setUsername(String uname) {
+        username.clear();
+        username.sendKeys(uname);
```

Comment:
daca nu mai folosesti LoginPage o poti sterge

Reply:
LoginPage il folosesc in LoginSteps